docs/learn/bossroom/getting-started-boss-room.md

```diff
@@ -6,200 +6,275 @@ description: Learn more about installing and running the Boss Room game sample.
-2. Save the binary as `Boss Room`.
+1. Check that you have installed the most recent [LTS editor version](https://unity.com/releases/2021-lts).
+    1. Include standalone support for **Windows/Mac** in your Unity Editor installation.
+2. Add the project to the Unity Hub by selecting the **Add** button and pointing it to the root folder of the downloaded project.
+    1. Unity imports all the project assets the first time you open the project, which takes longer than usual.
+3. Select the **Play** button. You can then host a new game or join an existing one using the in-game UI.
 
-After the build has completed, you can launch several instances of the built executable to both host and join a game.
+### Exploring the project
 
-:::important Mac Users
-To run multiple instances of the same app, you need to use the command line:
-1.  First, change your directory to the folder where you saved the Boss Room executable. For example, `cd Desktop/com.unity.multiplayer.samples.coop-<version>`.
-2.  Run the command `Open -n YourAppName.app`. If you saved the app as `BossRoom`, your command is `Open -n BossRoom.app`. However, if you saved your app as `Boss Room` with a space, your command needs to include quotation marks ("") around the executable name: `Open -n "BossRoom.app"`.
-:::
+Boss Room is an eight-player co-op RPG game experience, where players collaborate to fight imps, and then a boss. Players can select between classes that each have skills with didactically interesting networking characteristics. Control model is click-to-move, with skills triggered by a mouse button or hotkey.
 
-## Multiplayer over internet
+One of the eight clients acts as the host/server. That client will use a compositional approach so that its entities have both server and client components.
 
-In contrast to running a local setup, when playing over internet we do not necessarily need a built executable. You can run the game in editor.
+* The game is server-authoritative, with latency-masking animations.
+* Position updates are carried out through NetworkTransform that sync position and rotation.
+* Code is organized in domain-based assemblies. See our [ARCHITECTURE.md](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/blob/main/ARCHITECTURE.md) file for more details.
 
-Running the game over internet currently requires using [Port Forwarding](#port-forwarding) or [Integrating Boss Room with Unity Gaming Services](#integrating-boss-room-with-unity-gaming-services).
+### Testing multiplayer
 
-### Port Forwarding
+To see the multiplayer functionality in action, you can:
 
-The [Portforward Site](https://portforward.com/) has guides on how to enable port forwarding on a huge number of routers. Boss Room uses UDP and needs a 9998 external port to be open.
+* Run multiple instances of the game locally on your computer (using either ParrelSync or builds).
+* Connect to a friend over the internet.
```

Comment:
* Connect with a friend over your local area network
* Connect with a friend over the internet
  *  You can use UGS which helps remove the complexities of handling port forwarding
  *  Otherwise, the host will need to setup port forwarding on their router